pacman/utilities/algorithm_utilities/placer_algorithm_utilities.py

```diff
@@ -56,7 +57,7 @@ def get_same_chip_vertex_groups(graph):
         get_vertices_on_same_chip, graph=graph))
 
 
-def group_vertices(vertices, same_group_as_function):
+def group_vertices(vertices, same_group_as_function, cutoff=sys.maxsize):
```

Comment:
not in python doc

Reply:
should I reply picky or good spot? either way done.

Reply:
well here's another one. now that the ide isnt complaining about lack of python docs, its complaining that its never used....... so redundant param

Reply:
yet left over from when I was using my own function vs set builtins